app/views/publish/courses/_basic_details_tab.html.erb

```diff
@@ -139,7 +139,12 @@
 
       <% summary_list.row(html_attributes: { data: { qa: "course__start_date" } }) do |row| %>
         <% row.key { "Course starts" } %>
-        <% row.value { l(course.start_date&.to_date, format: :short) } %>
+        <% row.value do %>
+          <ul class="govuk-list">
```

Comment:
I think we want a `govuk-hint` class here

Reply:
Like this?
```
<%= l(course.start_date&.to_date, format: :short) %>
            <div class="govuk-hint">Academic year <%= course.cycle_range %></div>
```

Reply:
Have gone with
```
<p class="govuk-body"><%= l(course.start_date&.to_date, format: :short) %></p>
<p class="govuk-hint">Academic year <%= course.cycle_range %></p>
```

Reply:
Re testing, we can add to the spec/features/publish/viewing_a_course_details_spec.rb etc once we have the format nailed down